src/opensuse/42.3/Dockerfile

```diff
@@ -43,3 +95,6 @@ RUN zypper -n install --force-resolution \
 
 # Until OpenSuse.42.3 official packages are available, we have to restore the ubuntu ones instead.
 ENV __PUBLISH_RID=ubuntu.14.04-x64
```

Comment:
You may consider declaring these ENV within one Docker instruction to reduce the number of layers in the resulting image.
```
ENV env1=value1 \
        env2=value2
```

Reply:
Oh, I have not realized that even the env vars create a layer. Will do.